runtime/README.rst

```diff
@@ -3,35 +3,95 @@
 Catalyst Runtime
 ################
 
-The Catalyst Runtime is a C++ QIR runtime that enables the execution of Catalyst-compiled quantum programs, and is currently
-backed by state-vector simulators `PennyLane-Lightning <https://github.com/PennyLaneAI/pennylane-lightning>`_
+The Catalyst Runtime is a C++ QIR runtime that enables the execution of Catalyst-compiled
+quantum programs, and is currently backed by state-vector simulators
+`PennyLane-Lightning <https://github.com/PennyLaneAI/pennylane-lightning>`_
 and `Pennylane-Lightning-Kokkos <https://github.com/PennyLaneAI/pennylane-lightning-kokkos>`_.
-A complete list of the quantum instruction set supported by the runtime can be found in
+
+The runtime employs the ``QuantumDevice`` public API that enables the extension of backend quantum devices.
+This API accommodates two collections of abstract methods as follows:
```

Comment:
@maliasadi say I am interested in using this to create a runtime for my own device. Where can I read the `QuantumDevice`API documentation?

Reply:
Good point @josh146! We already generate the API, and here, I added a link to the `QuantumDevice` doc page. Let me know if this isn't sufficient.